latest/index.bs

```diff
@@ -175,7 +175,174 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
+        |       |             # The rows in each obsm array SHOULD be chunked the same as the rows in X.
+        |       ├── 0.0
+        │       │   ...
+        │       └── n.m
+        |
+        ├── varm              # You MAY add a varm group comtainer. The varm group contains arrays that annotate the columns in X.
+        |   │                 # The rows in each array MUST be index-matched to the columns in X (if present). 
+        |   |
+        │   ├── .zgroup
+        |   |
+        |   ├── .zattrs       # `.zattrs` MAY contain `"encoding-type"`, which is set to `"dict"` by AnnData.           
+        |   │                 # `.zattrs` MAY contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
+        |   │
+        │   └── varm_0        # You MAY add a zarr array for each varm matrix.
+        |       |             # Each varmm array MUST have the same number of rows as columns in X.
+        |       |             # The rows in each obsm array SHOULD be chunked the same as the columns in X.
+        |       ├── 0.0
+        │       │   ...
+        │       └── n.m
+        |
+        ├── obsp              # You MAY add a obsp group comtainer. The obsp group contains sparse arrays that annotate the rows in X.
+        |   │                 # The rows in each array MUST be index-matched to the columns in X (if present). 
+        |   |
+        │   ├── .zgroup
+        |   |
+        |   ├── .zattrs       # `.zattrs` MAY contain `"encoding-type"`, which is set to `"dict"` by AnnData.           
+        |   │                 # `.zattrs` MAY contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
+        |   │
+        │   └── obsp_0        # You MAY add a zarr group for each obsp array.
+        |       |             # Each obsp array MUST have the same number of rows as rows in X.
+        |       |
+        │       ├── .zgroup
+        |       |
+        |       ├── .zattrs   # `.zattrs` MAY contain `"encoding-type"`, which is set to `"csr_matrix"` by AnnData.           
+        |       │             # `.zattrs` MAY contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
+        |       |             # `.zattrs` MAY contain `"shape"` which is an array giving the shape of the densified array.
+        |       |
+        |       ├── data      # You MUST add a one-dimensional zarr array named "data". 
+        |       |   |         # `data` MAY be chunked as the user desires.
+        |       |   |
+        |       |   ├── 0
+        │       │   |   ...
+        │       |   └── n
+        |       |
+        |       ├── indices   # You MUST add a one-dimensional zarr array named "indices".
+        |       |   |         # `indices` MAY be chunked as the user desires.
+        |       |   |
+        |       |   ├── 0
+        │       │   |   ...
+        │       |   └── n
+        |       |
+        |       └── indptr    # You MUST add a one-dimensional zarr array named "indptr".
```

Comment:
@ivirshup, do you think we should provide more detail on what should be in `data`, `indices`, and `indptr`?

Reply:
Indices and indptr must be ints.

Were you thinking of more?

Reply:
Good point regarding indices and indptr.

I was also wondering if there is a spec we should reference or if saying csc/csr sparse matrix is enough for people to know what to put in data/indicies/indptr (I don't work with sparse matrices a ton myself)?

Reply:
Sorry, just seeing I didn't respond here!

We could [link out to wikipedia](https://en.wikipedia.org/wiki/Sparse_matrix#Compressed_sparse_row_(CSR,_CRS_or_Yale_format)) for a definition?

I think it would be hard to have a concise description that would explain it for someone who's totally unfamiliar, and a full description would be kinda long.

Maybe:

* `data`: array containing values of the non-zero entries
* `indices` column/ row indices of the non-zero entries (for CSR/ CSC respectively)
* `indptr` cumulative offsets for row/ columns indices (CSR/ CSC respectively)